sky/backends/backend_utils.py

```diff
@@ -1562,6 +1570,21 @@ def _process_cli_query(
         return []
 
     if returncode != 0:
+        if ('Unable to locate credentials. You can configure credentials by '
```

Comment:
nit: Should we have this case after L1570 for readability to be similar as what we do for the Azure?

Reply:
Good catch, done.